testsuite/cluster-test/src/cluster_builder.rs

```diff
@@ -521,7 +521,7 @@ impl ClusterBuilder {
                         format!("/ip4/{}/tcp/{}", validator_nodes[i].internal_ip, 6180).as_str(),
                     )
                     .expect("Failed to parse network address"),
-                    NetworkAddress::from_str(format!("/ip4/{}/tcp/{}", fullnode_ip, 6180).as_str())
```

Comment:
May I ask why do we change the port number?

Reply:
I found there is a public port 6182 in fullnode yaml, is this the reason we wanna make them consistency?

Reply:
it's not only about consistency, this config must be set correctly for the `StubbedNode` to properly connect to LibraNet via fullnode in the public network (otherwise it will try to connect to the validator's address... which won't work)

Reply:
I assume it's 3am now in Korean?..:)